Update one-logical-copy.md (#692)
MicrosoftDocs/fabric-docs: docs/real-time-analytics/one-logical-copy.md

    1. Select the **_delta_log** folder.
    1. Select a file to view the table metadata and schema. The editor that opens is in read-only format.

## Data types mapping

> ### Eventhouse to Delta parquet data types mapping

Reviewer: ```suggestion
### Event house to Delta parquet data types mapping
```

>  Eventhouse data types are mapped to Delta Parquet data types using the following rules:

Reviewer: ```suggestion
 Event house data types are mapped to Delta Parquet data types using the following rules. For more information on Event house data types, see [Scalar data types](/azure/data-explorer/kusto/query/scalar-data-types/index?context=/fabric/context/context-rta&pivots=fabric).
```

> | Eventhouse Data Type | Delta Data Type 

Reviewer: ```suggestion
| Event house Data Type | Delta Data Type 
```
